lib/config/options/index.ts

```diff
@@ -2057,7 +2057,7 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'insecureRegistry',
-    description: 'Explicitly turn on insecure Docker registry access (HTTP).',
+    description: 'Explicitly turn on unsafe Docker registry access (HTTP).',
```

Comment:
I think unsafe is easier to understand than insecure, but feel free to revert this with this suggestion:

```suggestion
    description: 'Explicitly turn on insecure Docker registry access (HTTP).',
```